tailwind.config.js

```diff
@@ -8,6 +8,11 @@ module.exports = {
       './pages/**/*.{js,ts,jsx,tsx}',
       './components/**/*.{js,ts,jsx,tsx}',
     ],
+    options: {
+      safelist: {
+        standard: ['outline-none'],
```

Comment:
Why is it needed to stop PurgeCSS from removing this class?

Reply:
🤔  No need, we can remove it!